### DISCUSSION TITLE
Failed scores in multiplayer still submit to leaderboards

### DISCUSSION THREAD

**Discussion by Flamiii** [ppy/osu#26367] [Q&A] [answered]
Map: https://lazer.ppy.sh/beatmapsets/881620#osu/1895402

I played this map in a multiplayer lobby and failed in the map, and my score still submitted to the leaderboard of the map when it shouldn't have. This has the potential to be game-breaking, since you can just play in a multiplayer lobby instead of using No Fail and completely bypass the 0.5x score multiplier you're supposed to have.

![osu_2024-01-03_09-40-29](https://github.com/ppy/osu/assets/74088315/448429de-9de0-4c3b-a6fc-0a6962cea9fa)

**Comment by frenzibyte** [comment, marked as answer]
https://github.com/ppy/osu/pull/26384
